plugins/database/src/db_plugin.cpp

```diff
@@ -3936,19 +3947,20 @@ irods::error db_reg_resc_op(
     cllBindVars[4] = resc_input[irods::RESOURCE_CLASS].c_str();
     cllBindVars[5] = resc_input[irods::RESOURCE_LOCATION].c_str();
     cllBindVars[6] = resc_input[irods::RESOURCE_PATH].c_str();
-    cllBindVars[7] = myTime;
-    cllBindVars[8] = myTime;
-    cllBindVars[9] = resc_input[irods::RESOURCE_CHILDREN].c_str();
-    cllBindVars[10] = resc_input[irods::RESOURCE_CONTEXT].c_str();
-    cllBindVars[11] = resc_input[irods::RESOURCE_PARENT].c_str();
-    cllBindVarCount = 12;
+    cllBindVars[7] = secs.c_str();
+    cllBindVars[8] = secs.c_str();
+    cllBindVars[9] = millis.c_str();
+    cllBindVars[10] = resc_input[irods::RESOURCE_CHILDREN].c_str();
+    cllBindVars[11] = resc_input[irods::RESOURCE_CONTEXT].c_str();
+    cllBindVars[12] = resc_input[irods::RESOURCE_PARENT].c_str();
+    cllBindVarCount = 13;
 
     if ( logSQL != 0 ) {
         log_sql::debug("chlRegResc SQL 4");
     }
-    status =  cmlExecuteNoAnswerSql(
-                  "insert into R_RESC_MAIN (resc_id, resc_name, zone_name, resc_type_name, resc_class_name, resc_net, resc_def_path, create_ts, modify_ts, resc_children, resc_context, resc_parent) values (?,?,?,?,?,?,?,?,?,?,?,?)",
-                  &icss );
+    status = cmlExecuteNoAnswerSql(
+        "insert into R_RESC_MAIN (resc_id, resc_name, zone_name, resc_type_name, resc_class_name, resc_net, resc_def_path, create_ts, modify_ts, modify_ts_millis, resc_children, resc_context, resc_parent) values (??,?,?,?,?,?,?,?,?,?,?,?)",
```

Comment:
i think the new column will be at the end?

Reply:
Order doesn't matter. The inputs simply need to be passed in the correct order.

Reply:
Oh?  I always thought inserts needed to be ordered.    Okay.

Ah, internet says if you name the variables, the values just have to match the named order.    If you don't name the variables, and only pass the values in, then the table order is what is followed.  Got it.